snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/cli/SnomedCommand.java

```diff
@@ -142,6 +146,79 @@ public void run(CommandLineStream out) {
 		}
 	}
 	
+	@picocli.CommandLine.Command(
+			name = "identifiers",
+			header = "Collects SNOMED CT core component identifiers",
+			description = "Collect SNOMED CT identifiers"
+			)
+	private static final class IdentifiersCommand extends Command {
+				
+		private static final Set<String> ALL_SCTID_STATUSES = FluentIterable.from(IdentifierStatus.values()).transform(IdentifierStatus::getSerializedName).toSet();
+				
+		@Option(names = { "-s", "--status" }, description = "The SctId status to filter for. Can be Available, Reserved, Assigned, or Published.", defaultValue = "All", required = false)
+		String status="All";
+		
+		@Parameters(paramLabel = "PATH", description = "The absolute path of the output folder")
+		String path;
+		
+		
+		@Override
+		public void run(CommandLineStream out) {
+			ISnomedIdentifierService identifierService = getContext().service(ISnomedIdentifierService.class);
+			
+			if (!(identifierService instanceof DefaultSnomedIdentifierService)) {
+				out.println("Can't run command without DefaultSnomedIdentifierService");
```

Comment:
It would be better to print out something like `The current Component Identifier Service does not support listing all the IDs via this command`.